### PR TITLE
[Fix #183] Properly indent after on receive

### LINKED ISSUE #183: 'after' is indented too much on 'receive'
Currently, `default_formatter` formats this code…

```erlang
receive
    {'DOWN', Ref, _Type, Pid, _Reason} ->
        finished
after Timeout ->
        error
end.
```

…like this…

```erlang
    receive
        {'DOWN', Ref, _Type, Pid, _Reason} ->
            finished
        after Timeout ->
            error
    end.
```

I would like it to be formatted like this…

```erlang
    receive
        {'DOWN', Ref, _Type, Pid, _Reason} ->
            finished
    after Timeout ->
        error
    end.
```

### PR DESCRIPTION
**Pull request by elbrujohalcon**
Left some comments for discussion.

### CODE REVIEW

File: test_app/after/src/others.erl [hunk 1]
```diff
@@ -44,6 +44,27 @@ receive_expr() ->
     receive
         with ->
             {no, timeout}
+    end,
+    receive after 1000 ->
+        {only, timeout}
+    end,
```
Comment (elbrujohalcon) [inline test_app/after/src/others.erl:50]: @juanbono @diegomanuel: Do you prefer this to be formatted this way or is it better as follows?
```erlang
receive
after 1000 ->
    {only, timeout}
end,
```

File: test_app/after/src/others.erl [hunk 2]
```diff
@@ -44,6 +44,27 @@ receive_expr() ->
...
+                         long and strange,
+                         but,
+                         possible,
+                         timeout,
+                         expression,
+                         that,
+                         is,
+                         super,
+                         extensive) ->
+        yeah
```
Comment (elbrujohalcon) [inline test_app/after/src/others.erl:67]: Same comment as above…
```erlang
receive after …
```
…or…
```erlang
receive
after …
```
?

File: test_app/after/src/others.erl (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply test_app/after/src/others.erl:50]: I'd go with what is in `others.erl`. The other alternatives "look like" something's missing.

Comment (paulo-ferraz-oliveira) [reply test_app/after/src/others.erl:50]: To be clear:
```erlang
receive after 1000 ->
    {only, timeout}
end
```

File: test_app/after/src/others.erl (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply test_app/after/src/others.erl:67]: If you're going for "saving space" you should align it to the left, as much as possible, but I'd be contradicting myself if I said something different here.

File: test_app/after/src/others.erl (hunk 1 above)
Comment (juanbono) [reply test_app/after/src/others.erl:50]: I'm with Paulo, the first alternative looks nicer to me:
```erlang
receive
after 1000 ->
    {only, timeout}
end,
```

**Review by juanbono — APPROVED** [review]

**Pull request merged**